wolfssl/include.am

```diff
@@ -16,10 +16,16 @@ nobase_include_HEADERS+= \
                          wolfssl/certs_test.h \
                          wolfssl/test.h \
                          wolfssl/version.h \
-                         wolfssl/options.h \
                          wolfssl/ocsp.h \
                          wolfssl/crl.h
 
 noinst_HEADERS+= \
                          wolfssl/internal.h
 
+# For distro build don't install options.h.
```

Comment:
options.h is not included when building wolfSSL because autoconf is only one of several build systems we support.  That is, we can't assume autoconf in general, and if autoconf is being used then the library already knows everything it needs to.  What options.h is for, is when a user turns on or off a feature that isn't a default that changes a structure that they may use themselves.  Imagine the user turns on Hardware Accelerator X with --enable-x that adds some additional fields to each crypto structure.  The user probably doesn't know what CFLAG we've set when they --enable-x  when they build their application and include some of our public headers.  If they don't set the right FLAG they'll get build time errors, or worse, run time problems.  options.h takes care of all the right FLAGS for the user for their application when using autoconf.  On some other platforms the user may directly set CFLAGS through their IDE when building wolfSSL and their application to make sure they match.  But that's not convenient with autoconf where we've followed the existing pattern of --enable-y --disable-z.  options.h is convenient in that case.  I hope that makes sense.